#Starting point of vector
geometry_msgs/Point start_point

#End point of vector
geometry_msgs/Point end_point